src/harness/testcases/relinquishment_testcase.py

```diff
@@ -146,3 +146,33 @@ def test_WINFF_FT_S_RLQ_4(self):
     self.assertFalse('cbsdId' in response)
     self.assertTrue(response['response']['responseCode'] == 103 or
                     response['response']['responseCode'] == 105)
+
+  @winnforum_testcase
+  def test_WINFF_FT_S_RLQ_5(self):
+    """CBSD relinquishment request of nonexistent grant
+
+    CBSD Harness sends Relinquishment Request to SAS including grant
+    with valid CBSD ID and nonexistent Grant ID. The response should
+    be FAIL.
+    """
+
+    # Register the device
+    device = json.load(
+        open(os.path.join('testcases', 'testdata', 'device_a.json')))
+    self._sas_admin.InjectFccId({'fccId': device_a['fccId']})
+    request = {'registrationRequest': [device]}
+    response = self._sas.Registration(request)['registrationResponse'][0]
+    # Check registration response
+    self.assertEqual(response['response']['responseCode'], 0)
+    self.assertTrue('cbsdId' in response)
```

Comment:
I don't think this is strictly necessary. If 'cbsdId' is not in response, line 168 will cause an error that's almost identical to the one you will see from line 167.